mint/build/s3select/install.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+#
+#  Mint (C) 2020 Minio, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+MINIO_PY_VERSION="5.0.7"
```

Comment:
Can this be set to being the latest `minio_py` version?

Reply:
> Can this be set to being the latest `minio_py` version?

The latest version is already 5.0.7

Reply:
@vadmeste what @kannappanr meant is make sure to always use latest - this would mean that we are stuck to 5.0.7 release always.

Reply:
> @vadmeste what @kannappanr meant is make sure to always use latest - this would mean that we are stuck to 5.0.7 release always.

Done